applications/GeoMechanicsApplication/custom_conditions/README.md

```diff
@@ -0,0 +1,141 @@
+# Microclimate Boundary Condition
+A microclimate boundary condition refers to the specific set of environmental factors and conditions that influence the climate within a small, localized area, known as a microclimate. Unlike the broader regional or global climate, microclimates are unique climatic conditions that can vary significantly within a relatively small space.
+
+Microclimate boundary conditions are determined by factors such as:
+
+- Air temperature
+- Solar radiation
+- Humidity
+- Precipitation
+- Wind speed
+
+## Radiation
+The net radiation $R_n$ $\mathrm{[W/m^2]}$ is calculated by:
+
+$$ R_n = R^s + R^{l,ab} - R^{l,em} $$
+
+where
+
+- $R^s$ = in-coming short wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,ab}$ = absorbed long wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,em}$ = emitted long wave radiation $\mathrm{[W/m^2]}$
+
+The incoming short wave radiation is defined by
+
+$$ R^s = \left(1 - \alpha \right) R_{g} $$
+
+where
+
+- $\alpha$ = Albedo cover or vegetation coefficient  $\mathrm{[-]}$
+- $R_{g}$ = Short wave radiation  $\mathrm{[W/m^2]}$ 
+
+$R_g$ is a user difined parameter which is usually provided as a "Time-Radiation" table.
+
+The absorbed long wave radiation is:
+
+$$ R^{l,ab}=  \epsilon \sigma \left( T_{at} +273.15 \right)^4 $$
+
+where
+
+- $\epsilon$ = effective emissivity  $\mathrm{[-]}$
+- $\sigma$ = Stefan Boltzmann's constant  $\mathrm{[5.67 \cdot 10^{-8}  W/m^{2 \circ}C^4]}$
+- $T_{at}$ = atmospheric temperature $\mathrm{[^{\circ}C]}$
+
+And the emitted long wave radiation,
+
+$$ R^{l,em} = \sigma \left( T_{ss} +273.15 \right)^4 $$
```

Comment:
https://www.sciencedirect.com/topics/earth-and-planetary-sciences/net-radiation uses thermal emissivity of the surface and thermal emissivity of the near surface atmosphere. In your case it looks that an ideal black body emits. Is it a good enough assumption?

Reply:
If I remember correctly (but I would have to read back up on it), indeed both absorption and emission have to include the emissivity (the properties of absorption and emission are linked in that regard). Therefore I would expect the heat flow to be something along the lines of $\sigma \epsilon (T_{ss}^4 - T_{at} ^4)$.

It's a bit of the top of my head, but I think we indeed have to have another look at this.

Reply:
As far as I understand, $T_{ass}^4-T_{at}^4) is taken into account. They are considered as separate terms in the equation.

Reply:
Indeed, but in the way it is now, the term $T_{ss}$ is not multiplied with $\epsilon$, while I would expect this to be the case